pkg/mimirtool/commands/alerts.go

```diff
@@ -71,12 +70,22 @@ func (a *AlertmanagerCommand) Register(app *kingpin.Application, envVars EnvVarN
 	// Get Alertmanager Configs Command
 	getAlertsCmd := alertCmd.Command("get", "Get the Alertmanager configuration that is currently in the Grafana Mimir Alertmanager.").Action(a.getConfig)
 	getAlertsCmd.Flag("disable-color", "disable colored output").BoolVar(&a.DisableColor)
+	getAlertsCmd.Flag("address", "Address of the Grafana Mimir cluster; alternatively, set "+envVars.Address+".").Envar(envVars.Address).Required().StringVar(&a.ClientConfig.Address)
+	getAlertsCmd.Flag("id", "Grafana Mimir tenant ID; alternatively, set "+envVars.TenantID+".").Envar(envVars.TenantID).Required().StringVar(&a.ClientConfig.ID)
```

Comment:
[nit] Since there's a bit of repetition here between `getAlertsCmd`, `deleteCmd` and `loadalertCmd`, what if we register these flags in a for loop, something like (pseudocode):

```
for _, cmd := range []*kingpin.CmdClause{getAlertsCmd, deleteCmd, loadalertCmd } {
   cmd.Flag("address", ...)
   cmd.Flag("id", ...)
}
```

Reply:
I like that - more DRY. Done.